cli/ops/testing.rs

```diff
@@ -90,15 +85,3 @@ fn op_dispatch_test_event(
   sender.send(event).ok();
   Ok(())
 }
-
-#[op]
-pub fn op_print(
-  state: &mut OpState,
-  msg: String,
-  _is_err: bool,
-) -> Result<(), AnyError> {
-  let sender = state.borrow::<UnboundedSender<TestEvent>>().clone();
-  let msg = TestOutput::String(msg);
-  sender.send(TestEvent::Output(msg)).ok();
-  Ok(())
-}
```

Comment:
CI failed because this change to only use the pipes exposed that what we're doing here is racy. I'm not exactly sure yet how we can fix this.

Reply:
It's unfortunate, but I'm going to have to implement some manual flushing for these :/

Converted to draft

Reply:
Ok, this looks good now and seems to have been solved by doing a flush of the stdout and stderr pipes before sending certain events.